src/stirling/source_connectors/socket_tracer/bcc_bpf_intf/common.h

```diff
@@ -79,6 +79,8 @@ struct conn_id_t {
   int32_t fd;
   // Unique id of the conn_id (timestamp).
   uint64_t tsid;
+  // TODO(AdityaAtulTewari) This belongs in the Upid struct but that creates a rewrite.
```

Comment:
I think we should remove this TODO. There are definitely valid reasons for making the cgid more of a useful construct for the rest of Pixie, but I think that specific solution is going to require a more detailed proposal and discussion.